docs/_docs/04_reference/cli-options.md

```diff
@@ -1150,3 +1157,66 @@ When this flag is set, Terragrunt will output its logs in JSON format.
 **Environment Variable**: `TERRAGRUNT_TF_JSON_LOG` (set to `true`)
 
 When this flag is set, Terragrunt will wrap Terraform `stdout` and `stderr` in JSON log messages. Works only with `--terragrunt-json-log` flag.
+
+### terragrunt-provider-cache
```

Comment:
For each of these new provider caching flags, add a link to the dedicated provider caching docs page!

Reply:
Added. Thanks.